### DISCUSSION TITLE
How to get a working JSON file with the schema definition out of typebox

### DISCUSSION THREAD

**Discussion by lucashfreitas** [sinclairzx81/typebox#267] [Q&A] [answered]
Hello @sinclairzx81 ,

I am using type box to define schemas but will need to use it across typescript and also Python projects, so the raw JSON schema will be necessary.

Does this library have anything that allows us to extract the JSON string from typebox objects? E.g

![image](https://user-images.githubusercontent.com/11416392/198151146-841a3776-e170-4b3c-bfbd-1699d165aee0.png)

**Comment by sinclairzx81** [comment, marked as answer]
@lucashfreitas Hey Lucas ! :) 

TypeBox types are JSON Schema objects, So you can just `console.log()` them to print them out. For example.

```typescript
const T = Type.Object({
  x: Type.Number(),
  y: Type.Number(),
  z: Type.Number(),
})

console.log(JSON.stringify(T, null, 2))

// outputs -> {
//   "type": "object",
//   "properties": {
//     "x": {
//       "type": "number"
//     },
//     "y": {
//       "type": "number"
//     },
//     "z": {
//       "type": "number"
//     }
//   },
//   "required": [
//     "x",
//     "y",
//     "z"
//   ]
// }
```
Hope that helps!
S

**Comment by lucashfreitas** [comment]
Oh, so easy! :) Thank you!
